application-services/custom/custom-trigger-nats-rpc/res/configuration.yaml

```
@@ -0,0 +1,17 @@
+Writable:
+  LogLevel: "INFO"
+  Telemetry:
+    Interval: "0s" # Disabled since require MessageBus Trigger to be in use.
```

This needs to be applied to all PRs. 

```suggestion
    Interval: "0s" # Disabled, use non-zero duration to enable. MessageBus below must not be disable if Metrics are enabled,
```

---

Done